pkg/test/case.go

```diff
@@ -200,7 +203,11 @@ func (t *Case) Run(test *testing.T, tc *report.Testcase) {
 		}
 	}
 
-	t.CollectEvents(ns.Name)
+	if funk.Contains(t.Suppress, "events") {
```

Comment:
I think it might make sense to make a constant for "events" and other types of logs that can be suppressed later, then we have a list of possible values for the cmd parameter

Reply:
yeah... I has that initial thought  as well... on it!